src/utils/feature-flags.ts

```diff
@@ -8,8 +8,8 @@ declare const window: any
  * Uses "business-edit" project (per LD client id in config).
  */
 const defaultFlagSet: LDFlagSet = {
-  'alteration-ui-enabled': false,
-  'correction-ui-enabled': false
+  'alteration-ui-enabled': true,
+  'correction-ui-enabled': true
```

Comment:
Don't commit this change either :)

Reply:
these are one of the banes of my existence ;)